From daa3258516325731193ccde810d51275a58e4fe5 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:07:55 -0400
Subject: [usb] xhci: Use IS_ENABLED() macro

Message-id: <1410887463-20674-404-git-send-email-dzickus@redhat.com>
Patchwork-id: 95330
O-Subject: [RHEL7 PATCH 403/591] usb: xhci: Use IS_ENABLED() macro
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 961b3d0a990f8b1214f5425c4acbebbd0b777e36
Author: Fabio Estevam <fabio.estevam@freescale.com>
Date:   Thu May 8 19:25:57 2014 +0300

    usb: xhci: Use IS_ENABLED() macro

    Using the IS_ENABLED() macro can make the code shorter and easier to read.

    Signed-off-by: Fabio Estevam <fabio.estevam@freescale.com>
    Signed-off-by: Mathias Nyman <mathias.nyman@linux.intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/xhci.h b/drivers/usb/host/xhci.h
index 0d141ad..614b279 100644
--- a/drivers/usb/host/xhci.h
+++ b/drivers/usb/host/xhci.h
@@ -1721,8 +1721,7 @@ static inline int xhci_register_pci(void) { return 0; }
 static inline void xhci_unregister_pci(void) {}
 #endif
 
-#if defined(CONFIG_USB_XHCI_PLATFORM) \
- || defined(CONFIG_USB_XHCI_PLATFORM_MODULE)
+#if IS_ENABLED(CONFIG_USB_XHCI_PLATFORM)
 int xhci_register_plat(void);
 void xhci_unregister_plat(void);
 #else
-- 
1.7.1